girder/cumulus/server/models/volume.py

```diff
@@ -109,6 +111,16 @@ def append_to_log(self, user, id, record):
         self.update({'_id': ObjectId(id)}, {'$push': {'log': record}})
         send_log_notification('volume', volume, record)
 
+    def update_volume(self, user, volume):
```

Comment:
Should I be calling save in some places then? `self.update(volume)` was not clearing the volume's `clusterId` after a detach_complete (and so it couldn't be deleted)